src/rgw/rgw_rados.cc

```diff
@@ -3115,7 +3115,7 @@ class RGWDataNotifier : public RGWRadosThread {
   RGWDataNotifierManager notify_mgr;
 
   uint64_t interval_msec() override {
-    return cct->_conf->rgw_md_notify_interval_msec;
+    return cct->_conf->rgw_data_notify_interval_msec;
```

Comment:
yeah, John Spray described the changes on the ceph-devel list, see https://www.spinics.net/lists/ceph-devel/msg37602.html